dali/operators/reader/loader/numpy_loader.h

```diff
@@ -54,6 +54,36 @@ class NumpyParseTarget{
   }
 };
 
+struct NumpyFileWrapper {
+  Tensor<CPUBackend> image;
+  std::string filename;
+  bool fortan_order;
+
+  const TypeInfo& type() const {
+    return image.type();
+  }
+
+  void set_type(const TypeInfo& type_info) {
+    image.set_type(type_info);
+  }
+
+  const TensorShape<>& shape() const {
+    return image.shape();
+  }
+
+  void set_shape(const TensorShape<>& shape) {
+    image.Resize(shape);
+  }
```

Comment:
This smells of inefficiency - a single call to `Resize(shape, type)` results in fewer reallocations.

Reply:
removed, it was not used anyway